h2o-r/tests/testdir_javapredict/runit_PUBDEV_7185_binomial_gam_mojo_large.R

```diff
@@ -0,0 +1,56 @@
+setwd(normalizePath(dirname(
+  R.utils::commandArgs(asValues = TRUE)$"f"
+)))
+source("../../scripts/h2o-r-test-setup.R")
+
+test.binomial.gam.mojo <-
+  function() {
+    #----------------------------------------------------------------------
+    # Run the test
+    #----------------------------------------------------------------------
+    e <- tryCatch({
+      numTest = 1000 # set test dataset to contain 1000 rows
+      params_prob_data <- setParmsData(numTest) # generate model parameters, random dataset
+      modelAndDir<-buildModelSaveMojoGAM(params_prob_data$params) # build the model and save mojo
+      filename = sprintf("%s/in.csv", modelAndDir$dirName) # save the test dataset into a in.csv file.
+      h2o.downloadCSV(params_prob_data$tDataset[,params_prob_data$params$x], filename)
+      twoFrames<-mojoH2Opredict(modelAndDir$model, modelAndDir$dirName, filename) # perform H2O and mojo prediction and return frames
+      h2o.downloadCSV(twoFrames$h2oPredict, sprintf("%s/h2oPred.csv", modelAndDir$dirName))
+      h2o.downloadCSV(twoFrames$mojoPredict, sprintf("%s/mojoOut.csv", modelAndDir$dirname))
+      compareFrames(twoFrames$h2oPredict,twoFrames$mojoPredict, prob=1, tolerance = 1e-4)
+    }, error = function(x) x)
+    if (!is.null(e)&& (!all(sapply("wget", grepl, e[[1]]))))
+      FAIL(e)   # throw error unless it is the stupid wget error.
```

Comment:
what wget error? if this sometimes fails on network it should fail for real or try again, this is just hiding test failures

Reply:
I removed it.